src/main/groovy/io/seqera/wave/core/ContainerAugmenter.groovy

```diff
@@ -303,12 +305,13 @@ class ContainerAugmenter {
         return new Tuple2<String, Integer>(digest, size)
     }
 
-    protected String getFirst(value) {
+    protected static String getFirst(value) {
```

Comment:
Maybe we can rename `getFirst` to something more meaningful

Reply:
done